d2/src/main/java/com/linkedin/d2/discovery/stores/zk/SymlinkUtil.java

```diff
@@ -64,4 +64,9 @@ public static boolean containsSymlink(String path)
   {
     return (firstSymlinkIndex(path) < 0) ? false : true;
   }
+
+  public static boolean isSymlinkNodeOrPath(String nodeNameOrPath)
+  {
+    return nodeNameOrPath != null && nodeNameOrPath.indexOf(SYMLINK_PREFIX) >= 0;
```

Comment:
How about change to 
```
return nodeNameOrPath != null && nodeNameOrPath.contains(SYMLINK_PREFIX);
```

Reply:
Oh. Thanks for ur explanation.You are right, if the `SYMLINK_PREFIX` is the `char` , it's prefer to use `indexOf`